### PR TITLE
Update new-teams-known-issues.md

### PR DESCRIPTION
**Pull request by msandtorvMSFT**
Added a known issue about Home Page rendering in newly created Class Teams in Teams EDU on T2.1 desktop

### CODE REVIEW

**Review by msandtorvMSFT — COMMENTED** [review]
Please review this change and let me know if anything is unclear.

File: Teams/new-teams-known-issues.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ ms.localizationpriority: high
 - Detecting multiple languages for spellchecker isn't currently available. 
 - Scheduling a Teams Live Event will redirect to the classic Teams web experience currently.
 - Producing a Teams Live Event is not currently available; you will need to switch back to classic Teams.
+- Setting up the Home Page in a newly created Class team in Microsoft Teams for Education currently doesn't work on desktop.
```
Comment (JohanFreelancer9) [inline Teams/new-teams-known-issues.md:30]:
```suggestion
- Setting up the Home Page in a newly created Class team in Microsoft Teams for Education isn't currently available on desktop.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msandtorvMSFT, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by msandtorvMSFT** [comment]
Am I supposed to do some action here? If so, I apologize in advance for my ignorance, as this is my first commit (For next time I will also change to my v-msandtorv_microsoft GitHub account and probably use VS Code).

It has been days now and we have quite a few other changes planned, but wanted to see that the process worked first, so please let me know if there's anything I need to do, or if there's a faster way of getting our changes published @Joanne ***@***.***>

Thanks,

Magnus Sandtorv

________________________________

**Pull request merged**